Godeps/Godeps.json

```diff
@@ -1310,153 +1310,133 @@
 		},
 		{
 			"ImportPath": "github.com/docker/docker/api",
-			"Comment": "docs-v1.12.0-rc4-2016-07-15-7401-g4f3616fb1",
-			"Rev": "4f3616fb1c112e206b88cb7a9922bf49067a7756"
+			"Comment": "docs-v1.12.0-rc4-2016-07-15-9510-ga9fbbdc8d",
+			"Rev": "a9fbbdc8dd8794b20af358382ab780559bca589d"
```

Comment:
What effective docker version are we bumping to?  Has sig-node weighed in on this update yet?

Reply:
its 1.38.0 and see the comment below :) thanks @Random-Liu